components/ai_chat/core/browser/conversation_handler.h

```diff
@@ -457,8 +467,10 @@ class ConversationHandler : public mojom::ConversationHandler,
 
   base::ObserverList<Observer> observers_;
   mojo::ReceiverSet<mojom::ConversationHandler> receivers_;
-  // TODO(petemill): Rename to ConversationUIHandler
+  mojo::ReceiverSet<mojom::UntrustedConversationHandler> untrusted_receivers_;
   mojo::RemoteSet<mojom::ConversationUI> conversation_ui_handlers_;
+  mojo::RemoteSet<mojom::UntrustedConversationUI>
+      untrusted_conversation_ui_handlers_;
```

Comment:
ConversationHandler has to manage receiver and remote for each parent trusted webui and child untrusted webui